include/Core/Base/BaseModule.php

```diff
@@ -123,6 +123,8 @@
 	 */
 	abstract protected function get_success_message( $items_deleted );
 
+	abstract protected function get_cron_action_name();
```

Comment:
Let's rename this to just `get_cron_name`. Also add phpdoc block to this function.

Generally for inherited methods, we should add the phpdoc block in the base class and we can ignore it in child class unless we are doing something different.